submissions/jeffersonMormah/database/transportDatabase.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE ohonbatransport;
```

Comment:
Did you redo this task? this should have been worked on before now.

Reply:
No i didn't, i just added it to the current task since is based on the subject matter on transport. @adaugochi

Reply:
You already have it 
![image](https://user-images.githubusercontent.com/43061141/99820852-98c00b00-2b51-11eb-9271-2c28bf450971.png)

Reply:
Yes I have it already

Reply:
Should I remove?

Reply:
Yes. or is there a reason why you still doing it again?

Reply:
@Jeffbuchi